pkg/codegen/schema/docs_parser.go

```diff
@@ -100,7 +100,7 @@ func (shortcodeParser) parseShortcode(line []byte, pos int) (int, int, int, bool
 			if inName {
 				nameEnd = pos
 			}
-			text, pos = text[3:], pos+3
```

Comment:
This change doesn't look like it's a refactor. 
I would expect a second assignment statement after line 103, that updates the value of `text`.

Reply:
Oh, it's part of the same linter fixes.
The value of `text` is never used again after this point.
If this condition matches, we set pos, break out of the loop,
and return `nameStart, nameEnd, pos, isClose, true`.
Let me add a comment here.